types/asana/index.d.ts

```diff
@@ -2211,6 +2211,25 @@ declare namespace asana {
              * @return {Promise} The requested resource
              */
             updateTask(taskGid: string, data?: Tasks.UpdateParams, dispatchOptions?: any): Promise<Tasks.Type>;
+
+            // https://developers.asana.com/docs/get-tasks-from-a-section
+            // https://github.com/Asana/node-asana/blob/master/lib/resources/gen/tasks.js#L314-L333
+            /**
+             * * Get tasks from a section
+             *   * @param {String} sectionGid: (required) The globally unique identifier for the section.
+             *   * @param {Object} params: Parameters for the request
```

Comment:
```suggestion
             *   * @param {Object} [params]: Parameters for the request
```
keep consistent, e.g with `dispatchOptions` as optional param

Reply:
Updated in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62476/commits/c014532ffb4b3632937b4a4aed42da747d638d94